app/controllers/proposals_controller.rb

```diff
@@ -98,6 +99,13 @@ def proposal_params
                                      speakers_attributes: [:bio, :user_id, :id])
   end
 
+  def require_invite
+    unless current_user.invitations.where(state: 'pending').includes(proposal_id: @proposal.id)
```

Comment:
I haven't seen includes used this way, so I may be missing something, but maybe this should look more like:

```
current_user.invitations.where(state: 'pending', proposal_id: @proposal.id).first
```